test/CMakeLists.txt

```diff
@@ -872,3 +872,40 @@ endif ()
 if (HDF5_TEST_API)
   add_subdirectory (API)
 endif ()
+
+# Build suffix connector
+set (HDF5_SUFFIX_PT_SOURCES
+  suffix_pt_vol_connector.c
+)
+
+set(HDF5_SUFFIX_PT_HEADERS
+  suffix_pt_vol_connector.h
+)
+
+add_library(suffix_pt_vol_connector
+  SHARED
```

Comment:
Just as an example of the build system complexity you have to keep in mind, it would probably be surprising here if somebody gets a shared library built when they are building the library with static libs only. And then if we skip building this VOL for static lib only builds of the library, we get less testing as opposed to the case where the VOL is built into the code and the tools can load and use it in either case.